plugins/kubernetes/app/models/kubernetes/template_filler.rb

```diff
@@ -359,6 +375,11 @@ def static_env
       env.merge!(Samson::Hooks.fire(:deploy_group_env, project, @doc.deploy_group).inject({}, :merge!))
     end
 
+    def blue_green_color
+      return @blue_green_color if defined?(@blue_green_color)
```

Comment:
`return if !@doc.kubernetes_role.blue_green`

Reply:
added that logic into release_doc itself